components/espace-agent-components/documents/document-wall.tsx

```diff
@@ -0,0 +1,106 @@
+import { PropsWithChildren } from 'react';
+import routes from '#clients/routes';
+import ButtonMonComptePro from '#components-ui/button-mon-compte-pro';
+import { Section } from '#components/section';
+import { FullTable } from '#components/table/full';
+import { EAdministration } from '#models/administrations/EAdministration';
+import { IUniteLegale } from '#models/index';
+
+const DocumentWall: React.FC<
+  PropsWithChildren<{
+    isVisible: boolean;
+    title: string;
+    id: string;
+    uniteLegale: IUniteLegale;
+  }>
+> = ({ uniteLegale, id, title, isVisible = false, children }) => {
+  if (!isVisible) {
+    return (
+      <Section title={title} id={id} sources={[EAdministration.INPI]}>
```

Comment:
I would keep the pink border for coherence. 
```suggestion
      <Section isProtected title={title} id={id} sources={[EAdministration.INPI]}>
```